crates/catalog/rest/testdata/rest_catalog/docker-compose.yaml

```diff
@@ -20,7 +20,7 @@ networks:
 
 services:
   rest:
-    image: apache/iceberg-rest-fixture
+    image: apache/iceberg-rest-fixture:1.9.2
```

Comment:
i think this is due to the new default behavior for iceberg-rest-fixture 

https://github.com/apache/iceberg/pull/13599